docs/Developer Documentation/ResumeController/index.md

```diff
@@ -0,0 +1,227 @@
+# Resume Controller
+
+This page will descrive internal structure and detailed design of Resume controller 
+
+|||
+Figure 1: ResumeController Overview
+![ResumeControllerClasses](./assets/resume_controller_classes.png)
+|||
+
+!!! NOTE
+Classes named like *Impl only _represent_ implementations of the abstract sub classes and may not be named the same in the SDL Core project.
+UML Refresher
+
+* Aggregation: Solid line with open diamond
+* Composition: Solid line with filled diamond
+* Inheritance: Dotted line with open arrow
+* Dependency: Dotted line with two prong arrow
+!!!
+
+## Resume Controller
+
+Resume controller responsibility is to cover Resumption requirements of SDL.
+There are 2 resumption types :
+
+ * HMI state resumption
+ * Data Resumption
+
+Resume controller do both.  
+
+### HMI state resumption
+
+In case of unexpected disconnect SDL should store application hmi state next 3 ignition cycles.
+On next application registration SDL should restore last saved application hmi_state.
+
+ResumptionData interface contains responsible for keeping application information. 
```

Comment:
wrong English grammar in
`ResumptionData interface contains responsible for keeping application information.`

Reply:
Fixed